### PR TITLE
fix(bingx) - migrate to ticksize

### PR DESCRIPTION
**Pull request by ttodua**

### CODE REVIEW

**Comment by ttodua** [comment]
@carlosmiei unrelated issue

File: ts/src/bingx.ts [hunk 1]
```diff
@@ -646,13 +654,13 @@ export default class bingx extends Exchange {
         const quote = this.safeCurrencyCode (quoteId);
         const currency = this.safeString (market, 'currency');
         const settle = this.safeCurrencyCode (currency);
-        let pricePrecision = this.safeInteger (market, 'pricePrecision');
+        let pricePrecision = this.safeNumber (market, 'tickSize');
```
Comment (carlosmiei) [inline ts/src/bingx.ts:657]: why did we change the order of the ifs here? we were reading pricePrecision first and then tickSize

Comment (ttodua) [reply ts/src/bingx.ts:657]: yes, i have changed it, because it shouldnt matter

Comment (ttodua) [reply ts/src/bingx.ts:657]: but i'll revert if you think we should retain same flow.

**Comment by ttodua** [comment]
@carlosmiei ^

**Pull request merged**